src/ray/protobuf/gcs.proto

```diff
@@ -301,6 +300,37 @@ message GcsNodeInfo {
   uint64 end_time_ms = 24;
 }
 
+message JobsAPIInfo {
+  // The status of the job.
+  string status = 1;
+  // The entrypoint command for this job.
+  string entrypoint = 2;
+  // A message describing the status in more detail.
+  string message = 3;
+  // TODO(architkulkarni): Populate this field
+  // Error type (e.g. Runtime env setup failure, Internal error, user script error)
+  string error_type = 4;
```

Comment:
should this be an `enum` rather than a string?

Reply:
Possibly, right now in the Python `JobInfo` it's a string, but it isn't even populated anywhere yet. Unless we need to finalize this decision now (for protobuf backwards compatibility reasons?) I think we can just leave it as string

Reply:
assuming this is only used internally in ray there's no compatibility issues to worry about